docs/dsp-tools-create.md

```diff
@@ -1191,32 +1066,43 @@ The following parent predefined resources are provided by DSP:
 
 `"cardinalities": [...]`
 
-Cardinalities is an array that contains the information about the connections between resources. It tells what type of
-connections a single resource has as well as how many times the connection is established. Thus, the actual "network" is
-saved in this array.
+An array that contains information about the relation between resources and properties. It tells what properties a
+resource can have as well as how many times the relation is established.
 
 - `cardinalities`: Array of references to the properties that the resource may hold including the cardinality. A
   cardinality has the following properties:
-    - `propname`: The name of the property. If it's used in the form ":"propname, the current ontology is referenced. If
-      the ":"is omitted, a DSP standard ontology is referenced, otherwise the full prefix of the ontology has to be
-      used.
-    - `gui_order`: An integer number which will help the GUI to display the properties in the desired order
+    - `propname`: The name of the property. If it's used in the form `:my_propname`, the current ontology is referenced.
+      Otherwise, the prefix of the ontology the property is part of has to be used.
+    - `gui_order`: An integer number which will help the GUI to display the properties in the desired order (optional)
     - `cardinality`: Indicates how often a given property may occur. The possible values are:
         - `"1"`: exactly once (mandatory one value and only one)
         - `"0-1"`: The value may be omitted, but can occur only once.
         - `"1-n"`: At least one value must be present, but multiple values may be present.
         - `"0-n"`: The value may be omitted, but may also occur multiple times.
 
+##### Comments
+
+`"comments": { "<lang>": "<comment>", "<lang>": "<comment>", ... }`
+
+Comments with language tags. The `comments` element is optional. Currently, "de", "en", "fr" and "it" are supported.
+
 Example for a resource definition:
 
 ```json
 {
   "resources": [
     {
       "name": "Schule",
-      "super": "Resource",
       "labels": {
-        "de": "Schule"
+        "de": "Schule",
+        "en": "School",
+        "fr": "Ecole"
+      },
```

Comment:
Above it it mentioned that German, English, French and Italian is supported in labels. It would be great, if you also added the "it" here.